src/routes/news/containers/NewsController.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { CityModel } from '@integreat-app/integreat-api-client'
+import type { ReceivedAction } from 'redux-first-router'
+import { redirect } from 'redux-first-router'
+import type { Dispatch } from 'redux'
+import type { StateType } from '../../../modules/app/StateType'
+import type { StoreActionType } from '../../../modules/app/StoreActionType'
+import { TUNEWS_ROUTE } from './../../../modules/app/route-configs/TunewsRouteConfig'
+import { TUNEWS_DETAILS_ROUTE } from './../../../modules/app/route-configs/TunewsDetailsRouteConfig'
+import { CATEGORIES_ROUTE } from './../../../modules/app/route-configs/CategoriesRouteConfig'
+import { LOCAL_NEWS_ROUTE } from './../../../modules/app/route-configs/LocalNewsRouteConfig'
+import { LOCAL_NEWS_DETAILS_ROUTE } from './../../../modules/app/route-configs/LocalNewsDetailsRouteConfig'
+
+type PropsType = {|
+  type: string,
+  city: string,
+  language: string,
+  cities: Array<CityModel>,
+  redirect: ReceivedAction => void,
+  children: React.Node
+|}
+
+export class NewsController extends React.Component<PropsType> {
+  componentDidMount () {
+    this.handleRedirect()
+  }
+
+  handleRedirect = () => {
```

Comment:
Did you remove it completely? If so, why?

Reply:
Yeah I removed it. I think it make more sense to show not found error instead of redirecting the user to another page. (I'll make some changes and push it)
before => if the user enters tunews link directly and tunews are disabled for this city it will redirect instead of showing not found error
@klinzo What do you think?